### PR TITLE
Add drills to measures on Overview dash

### PR DESCRIPTION
**Pull request by AnnaSerova**
#11

### CODE REVIEW

File: kpis_period_over_period.view.lkml [hunk 1]
```diff
@@ -0,0 +1,576 @@
+ view: kpis_period_over_period {
+   derived_table: {
+     datagroup_trigger: etl_datagroup
+     sql:
+    with this_quarter_metrics as (
```
Comment (jdavidson) [inline kpis_period_over_period.view.lkml]: Can we move this to a native DT?

File: kpis_period_over_period.view.lkml [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+    sql: CASE
+          WHEN {% parameter timeframe_filter %} = 'Quarter' THEN ${last_quarter_total_cost}
+          WHEN {% parameter timeframe_filter %} = 'Month' THEN ${last_month_total_cost}
+          WHEN {% parameter timeframe_filter %} = 'Week' THEN ${last_week_total_cost}
+          ELSE NULL
+          END ;;
+    value_format_name: usd_0
+  }
+
+  measure: avg_conversion_rate {
```
Comment (jdavidson) [inline kpis_period_over_period.view.lkml]: i think this is going to be an average of averages and not the avg conversion rate. you'll likely need to recreate the logic to make conversion rates - ${total_conversions} / ${total_clicks}

File: kpis_period_over_period.view.lkml [hunk 3]
```diff
@@ -0,0 +1,576 @@
...
+    sql: CASE
+          WHEN {% parameter timeframe_filter %} = 'Quarter' THEN ${last_quarter_conversions}
+          WHEN {% parameter timeframe_filter %} = 'Month' THEN ${last_month_conversions}
+          WHEN {% parameter timeframe_filter %} = 'Week' THEN ${last_week_conversions}
+          ELSE NULL
+          END ;;
+    value_format_name: decimal_0
+  }
+
+  measure: avg_cost_per_conversion {
```
Comment (jdavidson) [inline kpis_period_over_period.view.lkml]: weighted avg here too

File: kpis_period_over_period.view.lkml [hunk 4]
```diff
@@ -0,0 +1,576 @@
...
+          WHEN {% parameter timeframe_filter %} = 'Quarter' THEN ${this_quarter_avg_cost_per_conversion}
+          WHEN {% parameter timeframe_filter %} = 'Month' THEN ${this_month_avg_cost_per_conversion}
+          WHEN {% parameter timeframe_filter %} = 'Week' THEN ${this_week_avg_cost_per_conversion}
+          ELSE NULL
+          END ;;
+    value_format_name: usd
+  }
+
+  measure: avg_cost_per_conversion_last_period {
+    type: average
```
Comment (jdavidson) [inline kpis_period_over_period.view.lkml]: all the rest below here as well

File: kpis_period_over_period.view.lkml (hunk 1 above)
Comment (jdavidson) [reply kpis_period_over_period.view.lkml]: Do we need to create every DT or can we dynamically create a native DT with a parameterized dimension that would be this period or last period and the period could be a parameter. happy to give you more thoughts live.

**Comment by AnnaSerova** [comment]
While I'm still working on the period over period NDT, I updated this PR to add drill links to measures

**Pull request merged**